tests/blueprint_specific_tests/admin_bp_tests/test_admin_endpoints.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for endpoints in the auth blueprint
+"""
+import pytest
+from urllib.parse import urlparse
+from flask import url_for
+
+
+def test_admin_route(test_client):
+    """
+    GIVEN calling the route "/admin"
+    WHEN no user is logged in
+    THEN should return success code
+    """
+    res = test_client.get("/admin")
+    assert res.status_code == 200
```

Comment:
shouldn't `/admin` return a 401 when no user is logged in?

Reply:
Yes it should. I'll update the test but there's also some implementation needed in general for the admin functionality